tests/test_06_l2vpn_return_codes.py

```diff
@@ -273,7 +271,7 @@ def test_050_create_l2vpn_with_min_bw_out_of_range_code410(self):
         response = requests.post(api_url, json=payload)
         assert response.status_code == 410, response.text
```

Comment:
@gretelliz can you please change the return code here to 400? Because since the request has an invalid value (101, which out of the range), the correct return code should be indeed 400. You can create another test to validate 410 by created a few L2VPNs before which will consume the available bw and then request for a circuit which cannot be fulfilled on the current topology